src/components/measuresDetails.jsx

```diff
@@ -0,0 +1,109 @@
+import { graphql } from "gatsby";
+import React from "react";
+
+import { TotalAmountPanelItem } from "../components/totalAmountItem";
+import LandingPageHero from "../components/landingPageHero";
+import Layout from "../components/layout";
+import SideNav from "../components/sideNav";
+import SectionHeader from "../components/sectionHeader";
+import styles from "../templates/candidate.module.scss";
+import useWindowIsLarge from "../common/hooks/useWindowIsLarge";
+import VotersEdgeIcon from "../../static/images/votersEdge.png";
+import ChartSection from "./ChartSection";
+
+const supportingCommittees = [
+  {
+    title: "Yes on Measure E! San José Neighbors for Parks & People",
+    value: 500000,
+  },
+  {
+    title: "Yes on Measure E! San José Neighbors for Parks & People",
+    value: 400000,
+  },
+  {
+    title: "Yes on Measure E! San José Neighbors for Parks & People",
+    value: 14000,
+  },
+];
+
+
+function MeasureDetails(props) {
```

Comment:
nit: should we rename the file `measureDetails` to match? (since it seems like it's details for a specific measure)